tsdb/config.go

```diff
@@ -98,8 +99,14 @@ type Config struct {
 }
 
 func NewConfig() Config {
+	defaultEngine := DefaultEngine
+	engine := os.Getenv("INFLUXDB_ENGINE")
```

Comment:
I was wondering about that! I had this vague memory that we parsed environment overrides, with names dictated by the configs.

Let me try that.